controller/src/main/java/org/jboss/as/controller/AttributeParser.java

```diff
@@ -89,10 +92,14 @@ private ModelNode parse(final AttributeDefinition attribute, final String value)
         final String trimmed = value == null ? null : value.trim();
         ModelNode node;
         if (trimmed != null) {
-            if (attribute.isAllowExpression()) {
+            if (attribute.isAllowExpression() && ParseUtils.isExpression(trimmed)) {
                 node = ParseUtils.parsePossibleExpression(trimmed);
```

Comment:
This (and the others) can just be

```
node = new ModelNode(new ValueExpression(value));
```

That's all that parsePossibleExpression does, except for an isExpression() call that's now redundant.